changelog.dd

```diff
@@ -45,6 +45,8 @@ $(BUGSTITLE Library Changes,
         $(XREF conv, to) instead.)
     $(LI $(RELATIVE_LINK2 min-max-element,
     `std.algorithm.searching.{min,max}Element` for ranges have been added.))
+    $(LI $(RELATIVE_LINK2 quantize, Added $(XREF math, quantize), for rounding to
```

Comment:
You can run dmd -D with file yourself. 
However it's planned that DAutoTest also shows the changelog. This will most likely happen after the next release - once we moved to have a changelog folder with the pending entries. 
Ping @CyberShadow